HTMLFileSetUtils.spec

```diff
@@ -1,7 +1,50 @@
 /*
-A KBase module: HTMLFileSetUtils
+A utility for uploading HTML file sets to the KBase data stores.
 */
 
 module HTMLFileSetUtils {
 
+	/* A HTML file set as saved in the Workspace Service.
+		file - a base64 encoded zip file containing HTML files.
+	 */
+	typedef structure {
+		string file;
```

Comment:
The service does exactly that. This is the workspace type, where the data is stored, so this is an accurate description of the data. The service API completely encapsulates this type.

It's the actual file, which is why I called it file.